sync_diff_inspector/source/source.go

```diff
@@ -269,22 +270,36 @@ func initDBConn(ctx context.Context, cfg *config.Config) error {
 	}
 	// we had `cfg.SplitThreadCount` producers and `cfg.CheckThreadCount` consumer to use db connections maybe and `cfg.CheckThreadCount` splitter to split buckets.
 	// so the connection count need to be cfg.SplitThreadCount + cfg.CheckThreadCount + cfg.CheckThreadCount.
-	targetConn, err := common.ConnectMySQL(cfg.Task.TargetInstance.ToDriverConfig(), cfg.SplitThreadCount+2*cfg.CheckThreadCount)
+	targetConn, err := common.ConnectMySQL(
+		&cfg.Task.TargetInstance.SessionConfig,
+		cfg.Task.TargetInstance.ToDriverConfig(),
+		cfg.SplitThreadCount+2*cfg.CheckThreadCount,
+	)
 	if err != nil {
+		log.Error(fmt.Sprintf("failed to configure session for data source '%s'", cfg.Task.Target),
+			zap.String("error", err.Error()),
+		)
```

Comment:
```suggestion
		log.Error("failed to configure session", zap.String("data-source", cfg.Task.Target), zap.Error(err))
```

`printf` should be avoided when using structured logging